pallets/gear/src/internal.rs

```diff
@@ -29,35 +30,44 @@ use common::{
         MessageWaitedSystemReason::ProgramIsNotInitialized, MessageWokenReason, Reason, Reason::*,
         UserMessageReadReason, UserMessageReadRuntimeReason,
     },
-    gas_provider::{GasNodeId, GasNodeIdOf, Imbalance},
+    gas_provider::{GasNodeId, Imbalance},
     scheduler::*,
     storage::*,
-    GasPrice, GasTree, Origin,
+    GasPrice, GasTree, LockId, LockableTree, Origin,
 };
 use core::cmp::{Ord, Ordering};
 use core_processor::common::ActorExecutionErrorReason;
-use frame_support::{
-    codec::{Decode, Encode},
-    traits::{BalanceStatus, Currency, ExistenceRequirement, ReservableCurrency},
-};
+use frame_support::traits::{BalanceStatus, Currency, ExistenceRequirement, ReservableCurrency};
 use frame_system::pallet_prelude::BlockNumberFor;
 use gear_core::{
     ids::{MessageId, ProgramId, ReservationId},
     message::{Dispatch, DispatchKind, Message, StoredDispatch, StoredMessage},
 };
 use sp_runtime::traits::{Get, One, SaturatedConversion, Saturating, UniqueSaturatedInto, Zero};
 
-/// Cost builder for `HoldBound<T>`.
-#[derive(Clone, Debug, Decode, Encode, PartialEq, Eq, PartialOrd, Ord)]
-pub(crate) struct HoldBoundCost<T: Config>(SchedulingCostOf<T>);
+/// `HoldBound<T>` builder
+#[derive(Clone, Debug)]
+pub(crate) struct HoldBoundBuilder<T: Config> {
+    storage_type: StorageType,
+    cost: SchedulingCostOf<T>,
+}
 
 #[allow(unused)]
```

Comment:
I guess it's obsolete?

Reply:
Seems, there still is a currently unused method in the `HoldBoundBuilder` implementation. Hence leaving as is for now.